### PR TITLE
Add Zomato

### PR DESCRIPTION
**Pull request by saviokay**
Zomato, a popular Indian restaurant search and discovery web service. In 2017, the credentials of millions of its users were hacked and was known as one of the largest data leaks in Indian history. Even after this event, the service hasn't implemented any concert measures to avoid that incident from occurring again. The service is known to be used by the bulk of the population daily.  I have included Zomato to the Food list to help push the service to understand about a safe internet and of dependable practices.

(`https://economictimes.indiatimes.com/small-biz/security-tech/security/zomato-hacked-security-breach-results-in-17-million-user-data-stolen/articleshow/58729251.cms`)

### CODE REVIEW

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request @saviokay,
the image needs to be 32x32 pixels. 
Try this one:
zomato

**Comment by saviokay** [comment]
@Carlgo11 Sure Thing!. Thanks for the comment.
I'll make the required changes.

File: _data/food.yml [hunk 1]
```diff
@@ -34,3 +34,10 @@ websites:
       tfa: No
       twitter: tacobell
       facebook: tacobell
+ 
+    - name: Zomato
+      url: https://www.zomato.com
+      img: zomato.png
+      tfa: No
+      twitter: zomato
```
Comment (Carlgo11) [inline _data/food.yml]: Just noticed that they spell their twitter handle with a capital Z.
```suggestion
      twitter: Zomato
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by saviokay** [comment]
I have made all the required changes including the suggestions. Kindly review the pull request.

**Comment by Carlgo11** [comment]
@saviokay you did but you also reverted those changes in 9876ed1

**Comment by saviokay** [comment]
Apologies @Carlgo11 . I have made the corrections.

**Review by Carlgo11 — APPROVED** [review]

**Comment by saviokay** [comment]
@Carlgo11 If everything seems okay. Can we approve this pull request and merge it?
Thank you.

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**